CMakeLists.txt

```diff
@@ -109,6 +109,7 @@ else()
     if( "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" )
         if( CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 4.0.0 OR CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 4.0.0 )
             set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-invalid-partial-specialization" )
+            set( CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -latomic")
```

Comment:
Not clear to me if `target_link_libraries( eosio_chain PRIVATE -latomic)` in `libraries/chain/CMakeLists.txt` would be better.

Reply:
That would probably be more idiomatic, especially as long as we're just linking libatomic and not doing more hairy linker commands. Wouldn't it need to be `PUBLIC` though :thinking: